docs/source/users/install_instructions/externals-matlab.rst

```diff
@@ -0,0 +1,130 @@
+.. _external-instructions-matlab:
+
+####################################################
+Installing Matlab through WSL in a Linux environment
+####################################################
+
+There are a couple of ways to install MATLAB through WSL in a Linux environment, one of which requires X11 Forwarding, 
+while the other doesn't require a graphical interface.
+
+
+.. _with_x11_forwarding:
+
+Method 1: through X11 Forwarding
+********************************
+
+X11 Forwarding can be enabled through a server like VcXsrv, an X server for Windows that allows users to run graphical apps 
+from a Linux environment on a Windows  machine. 
+
+Enabling X11 Forwarding
+-----------------------
+
+Given VcXsrv is installed, the following steps are necessary to enable it:
+
+#. Start XLaunch.
+#. Click "Next" till the window "Extra settings".
+#. In the window "Extra settings", make sure the box "Disable access control" is ticked. 
+#. Click "Next" and then "Finish".
+
+For X11 Forwarding to work, it is also necessary to set a couple of environment variables (this could be done through `~./bashrc`, 
+to make sure there is no need to repeat the process in the future). Specifically, the environment variable called ``DISPLAY`` can be 
+set by running, e.g., ``export DISPLAY=:0``, and the environment variable ``LIBGL_ALWAYS_INDIRECT`` must be set to 1.
+
+At this point, X11 Forwarding can be tested by running ``xeyes`` from the WSL Linux environment, which should cause a pop-up with a 
+pair of eyes to appear. If ``xeyes`` does not work, it will be necessary to follow these steps:
+
+#. Navigate to Control Panel > System and Security > Defender Firewall > Advanced settings > Inbound rules. 
+#. Here, there should be two rules with the name "VcXsrv windows server" marked as Public. If these have red stop signs next to them, 
+   click on them and "allow" the connection. This operation requires admin rights. 
+
+Once this has been done, running ``xeyes`` should work and open the pop-up. 
+
+
+Installing MATLAB
+-----------------
+
+Having configured the X11 Forwarding, this can be used for installing MATLAB. For this, one should follow the instructions 
+in `<https://uk.mathworks.com/help/install/ug/install-products-with-internet-connection.html>`_. It is possible to check that the 
+version of MATLAB being downloaded is compatible with the Python version being used, by referring to 
+`<https://uk.mathworks.com/support/requirements/python-compatibility.html>`_. 
+
+After running ``sudo ./install`` from the MATLAB folder (as suggested in the MathWorks instructions), the graphical interface 
+will open. Here, the user is asked to select a suitable folder for the MATLAB installation. Let's call this path `<matlabroot>`. 
+In the last window, before the installation starts, there will be a question on whether to create symbolic links and where. 
```

Comment:
You probably want to put some guidance into what toolboxes are needed -- all of them is overkill, and takes forever. I think we just need MATLAB, Curve Fitting Toolbox, Optimization Toolbox, and Statistics and Machine Learning Toolbox.